napalm_junos/junos.py

```diff
@@ -1325,3 +1325,26 @@ def get_optics(self):
             optics_detail[intf_optic_item[0]] = intf_optics
 
         return optics_detail
+
+    def get_config(self, retrieve='all'):
+        rv = {
+            'startup': '',
+            'running': '',
+            'candidate': ''
+        }
+
+        options = {
+            'format': 'text',
+            'database': 'candidate'
+        }
+
+        if retrieve in ('candidate', 'all'):
+            config = self.device.rpc.get_config(filter_xml=None, options=options)
```

Comment:
I don't agree with that, if I call get_config(retrieve='candidate') I'd expect something back and not an empty string. Happy to do it if that's what you guys have decided.